starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,53 @@
 // Names and Input
 
+console.log("I'm Ready!");
+
+var hacker1 = "Heli"
+console.log("The driver's name is " + hacker1)
+
+var hacker2 = prompt("what's the navigator's name?")
+console.log("The navigator's name is " + hacker2)
 
 //Conditionals
 
+if ( hacker1.length > hacker2.length) {
+    console.log("The Driver has the longest name, it has" + hacker1.length +  "characters")
+  } else if ( hacker2.length > hacker1.length) {
+    console.log("Yo, navigator got the longest name, it has" + hacker2.length + "characters")
+  } else {
+    console.log("wow, you both got equally long names, " + hacker1.length + "characters!!")
+  }
+  
+  var string = " "
+  
+  for (var i = 0; i < hacker1.length; i++) {
+   string = string + hacker1[i].toUpperCase() + " "
+  }   
+   console.log(string)
+
+   var sentence1 = ["The driver's name goes first", "Yo, the navigator goes first definitely", "What?! You both got the same name?"]
+
+    console.log(sentence1.sort())
+
+// Palindromo
+
+function checkPalindrome(stringCheck){
```

Comment:
Very easy way to check if the given word is palindrome is using .split, .reverse, .join methods. Try something like this: 
```
var str = prompt("Let's check the palindrome. Give me a word:");

function checkPalindrome(str) {
  return (
    str ==
    str
      .split("")
      .reverse()
      .join("")
  );
}
var value = checkPalindrome(str);
// console.log("value:",value)

if (value === true) {
  console.log("The word is palindrome.");
} else {
  console.log("The word isn't palindrome.");
}
```